securedrop/tests/test_journalist.py

```diff
@@ -1207,9 +1206,7 @@ def test_journalist_session_expiration(self):
                         resp.data.decode('utf-8'))
         finally:
             if has_session_expiration:
-                config.SESSION_EXPIRATION_MINUTES = old_expiration
```

Comment:
The deletion of the the two lines below need to be explained in detail. In particular it should be clearly articulated in the commit message why these two lines become redundant and exactly which part of the code using it exist and how they are not impacted by this change. I'm sure you did your due diligence but it needs to be spelled out for the reader.